### PR TITLE
Add the Plywrks Ji-Eun

### PR DESCRIPTION
**Pull request by ramonimbao**
## Description

Adds the Plywrks Ji-Eun board.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plywrks/ji_eun/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/plywrks/ji_eun/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

File: keyboards/plywrks/ji_eun/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/plywrks/ji_eun/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/plywrks/ji_eun/keymaps/default/keymap.c (hunk 1 above)
Comment (ramonimbao) [reply keyboards/plywrks/ji_eun/keymaps/default/keymap.c:4]: Added the missing license headers in both files. Thanks!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**